Changelog.md

```diff
@@ -1,5 +1,8 @@
 WhateverGreen Changelog
 =======================
+#### v1.6.5
+- Revert the optimizations done by the compiler in backlight related functions to make Backlight Registers Fix (BLR) and Backlight Smoother (BLR) work on macOS 13.4. (by @0xFireWolf) 
```

Comment:
typo: Backlight **Smoother** should be BL**S**

Also, maybe worth mentioning the boot argument `-igfxbrs` that helps to enable this patch? Could potentially help users have better visibility I suppose. For eg: added boot argument `-igfxbrs` to revert optimizations done by the compiler in backlight related functions to make Backlight Registers Fix (BLR) and Backlight Smoother (BLS) work on macOS 13.4.

Reply:
I compiled this latest commit from 0xFireWolf:squashed using xcode 14.3
Successfully compiled but it is not work for me  
I m facing same backlight black screen issue during boot process
Please guide me i m running macOS 13.4 version
M i something wrong?

Reply:
> typo: Backlight **Smoother** should be BL**S**
> 
> Also, maybe worth mentioning the boot argument `-igfxbrs` that helps to enable this patch? Could potentially help users have better visibility I suppose. For eg: added boot argument `-igfxbrs` to revert optimizations done by the compiler in backlight related functions to make Backlight Registers Fix (BLR) and Backlight Smoother (BLS) work on macOS 13.4.

Thanks for the suggestions. Fixed.

Reply:
> I compiled this latest commit from 0xFireWolf:squashed using xcode 14.3 Successfully compiled but it is not work for me I m facing same backlight black screen issue during boot process Please guide me i m running macOS 13.4 version Enable backlihgt register fix already in config M i something wrong?

Please make sure that you have added the new boot argument `-igfxbrs` or device property `enable-backlight-registers-supplemental-fix`. Also, please avoid posting questions in a **code review thread** like this.